command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
+rucio did parent
```

Comment:
Why not have a `--parent` command-line option for `did list`?

Reply:
I think exclusively because they have different kwargs (parent uses --pfn and --guid, but list does not.) I wonder if that's a good way to handle it so that it's obvious these are only options when using --parent...

Reply:
I tested out both `--guid` and `--pfn` and none of them work. Very nice.

If these options *are* useful, I don’t see why we can’t implement them for regular listing of DIDs in the future.

Reply:
Cool, added `show --parent` as an option, and hid the guid and pfn options. 
I can make some issues cataloguing these proposed changes.